examples/functions-with-rest-parameter/functions_with_rest_parameter.bal

```diff
@@ -23,27 +23,27 @@ function main(string... args) {
     // Call the function by passing only the required parameter.
     printFruits(",");
 
-    // Call the function by passing the required parameter, and one value for 
-    // rest parameter.
+    // Call the function by passing the required parameter, and 
+    // a rest parameter that has one value.
```

Comment:
I feel "one value for the rest parameter" makes more sense here (similar to line 34). Since there can be only one rest parameter, and it can take multiple values..